docs/public/installation/prerequisites.md

```diff
@@ -35,13 +36,14 @@ at large cloud providers.
 
 Operating System | MariaDB | PostgreSQL
 ---------------- | --------| ---------------
-Debian 12        | 10.11   | 15.5
+Debian 12        | 10.11   | 15.6
 Docker           | n/a     | n/a
-FreeBSD 14.0     | 8.0 (*) | 15
-Rocky Linux 8.9  | 10.3    | 10.23
-Rocky Linux 9.3  | 10.5    | 13.13
-Ubuntu 20.04     | 10.3    | 12.17
-Ubuntu 22.04     | 10.6    | 14.10
+FreeBSD 14.1     | 8.0 (*) | 16.3
+Rocky Linux 8.10 | 10.11   | 16.1
+Rocky Linux 9.4  | 10.11   | 16.1
```

Comment:
In Rocky Linux 9.4, MariaDB and PostgreSQL versions installed by default by dnf are as follows:
```suggestion
Rocky Linux 9.4  | 10.5    | 13.14
```